docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -659,28 +713,24 @@ network="rinkeby"
 Predefined network configuration.
 The default is `mainnet`.
 
-Possible values are :
-
-`mainnet`
-:   Main Ethereum network
-
-`ropsten`
-:   PoW test network similar to current main Ethereum network. 
+Possible values are:
 
-`rinkeby`
-:   PoA test network using Clique.
+| Network   | Description                                                                   |
+|-----------|-------------------------------------------------------------------------------|
+| `classic` | Ethereum Classic network                                                      |
+| `dev`     | PoW development network with a very low difficulty to enable local CPU mining |
+| `goerli`  | PoA test network using Clique                                                 |
+| `kotti`   | Ethereum Classic PoA test network using Clique                                |
+| `mainnet` | Main Ethereum network                                                         |
+| `mordor ` | Ethereum Classic PoW test network                                             |
+| `rinkeby` | PoA test network using Clique                                                 |
+| `ropsten` | PoW test network similar to current main Ethereum network                     |
 
```

Comment:
Here also, sorted by family, then importance.  It's a subjective sort but I think we should reflect the hierarchy here.

Reply:
I added the Chain column and ordered them by importance, according to the list in one of the previous comments, and a Type column.